db/event_handler.go

```diff
@@ -58,9 +58,8 @@ func NewWebhook(url string, filterFnString string, timeout *uint64) (*Webhook, e
 
 	// Initialize transport and client
 	t := http.DefaultTransport.(*http.Transport)
-	transport := *t
-	transport.DisableKeepAlives = false
-	wh.client = &http.Client{Transport: &transport, Timeout: wh.timeout}
+	t.DisableKeepAlives = false
```

Comment:
Not too sure this is what we want here. I think we'd be modifying the stdlib `http.DefaultTransport` variable instead of doing the explicit copy we were doing before.

Reply:
As discussed, until we upgrade to a version of go that supports clone on transport, need to create a new transport and manually copy over the properites that are set in DefaultTransport.